stdlib-integration-tests/sql/tests/jdbc-integration-test.bal

```diff
@@ -25,13 +25,15 @@ string dbPath = checkpanic file:getAbsolutePath("target/databases");
 string jdbcUrl = "jdbc:h2:" + dbPath + "/" + "CONNECT_DB";
 
 @test:Config {
+    enable: false
```

Comment:
Rather than removing separately, shall we add this to the ignore list in the build task?